src/detect-xbits.c

```diff
@@ -365,6 +365,10 @@ int DetectXbitSetup (DetectEngineCtx *de_ctx, Signature *s, const char *rawstr)
             /* modifiers, only run when entire sig has matched */
             SigMatchAppendSMToList(s, sm, DETECT_SM_LIST_POSTMATCH);
             break;
+        default:
+            SigMatchFree(de_ctx, sm);
```

Comment:
more dead code, we can have an assertion here if we must. But I don't like adding code that isn't exercised at all just to please the static code analyzer